docs/guide/tutorials.rst

```diff
@@ -0,0 +1,244 @@
+	``RetentionTime,Annotation,PrecursorMz,MS1 Res,ProductMz,MS2 Res,Dwell,``
+	``Fragmentor,Collision Energy,Cell Accelerator Voltage,LibraryIntensity,decoy,``
+	``PeptideSequence,LabelType,PrecursorCharge,FragmentCharge,FragmentType,``
+	``FragmentSeriesNumber,quantifying_transition,identifying_transition,``
+	``detecting_transition``.
+	With a sample entry that may look like the following :
+	``Serotonin,,Serotonin,Serotonin,1.87,112.2,0,``
+	``Unit,0,Unit,,,,,1,0,,Light,1,1,,1,TRUE,FALSE,TRUE``.
+
+#. workflow.csv
+	The workflow steps, which are parsed by SmartPeak to process the data, 
+	are listed in this file under the column ``command_name``. 
+	A full list of the commands can be found in :ref:`Workflow Commands`.
+
+
+Targeted quantitation with HPLC data
+------------------------------------
+
+This tutorial walks you through the workflow for analyzing targeted HPLC data
+ starting from input file generation, to processing the data in SmartPeak, 
+ to reviewing the data in SmartPeak, to reporting the results for later use.
+
+Objectives
+~~~~~~~~~~
+
+#. Obtaining the SOP for the workflow.
+#. Choosing a data set for demonstrating the workflow.
+#. Creating an optimized SmartPeak input templates for running the workflow.
+
+The Workflows include
+~~~~~~~~~~~~~~~~~~~~~
+
+#. Calculating the calibration curves using Standards
+#. Processing Unknowns
+
+Steps
+~~~~~
+
+The tutorial includes the following steps :
+
+#. Setting up the input files
+
+The data set used can be found in 
+`HPLC_UV_Standards <https://github.com/AutoFlowResearch/SmartPeak/tree/develop/src/examples/data/HPLC_UV_Standards>`_ and
+`HPLC_UV_Unknowns <https://github.com/AutoFlowResearch/SmartPeak/tree/develop/src/examples/data/HPLC_UV_Unknowns>`_ 
+for the HPLC UV Standards and HPLC UV Unknowns respectively.
+
+#. Defining the workflow in SmartPeak
+
+For HPLC UV Standards analysis, the following steps are saved 
+into the ``workflow.csv`` file. Alternatively, steps can be replaced, 
+added or deleted direclty from SmartPeakGUI. 
+A detailed explanation of each command step
+can be found in :ref:`Workflow Commands`.
+
+	* LOAD_RAW_DATA
+	* MAP_CHROMATOGRAMS
+	* EXTRACT_CHROMATOGRAM_WINDOWS
+	* ZERO_CHROMATOGRAM_BASELINE
+	* PICK_MRM_FEATURES
+	* CHECK_FEATURES
+	* SELECT_FEATURES
+	* CALCULATE_CALIBRATION
+	* STORE_QUANTITATION_METHODS
+	* QUANTIFY_FEATURES
+	* STORE_FEATURES
+
+The workflow steps for HPLC UV Unknowns are :
+
+	* LOAD_RAW_DATA
+	* MAP_CHROMATOGRAMS
+	* EXTRACT_CHROMATOGRAM_WINDOWS
+	* ZERO_CHROMATOGRAM_BASELINE
+	* PICK_MRM_FEATURES
+	* QUANTIFY_FEATURES
+	* CHECK_FEATURES
+	* SELECT_FEATURES
+	* STORE_FEATURES
+
+#. Running the workflow in SmartPeak
+
+	To run the analysis, please follow the steps for 
+	:ref:`Using SmartPeak GUI` or :ref:`Using SmartPeak CLI`
+	to execute the workflow steps and review the results including plotting.
+
+#. Reporting the results
+
+	To export the results, please follow the :ref:`Export report`.
```

Comment:
It would be great to discuss the export options and formats available to the user.